daemon/main.c

```diff
@@ -921,8 +921,8 @@ int main(int argc, char **argv) {
                     {
                         char* stacksize_string = "stacksize=";
                         char* debug_flags_string = "debug_flags=";
-                        char* createdataset_string = "createdataset=";
-                        char* stresstest_string = "stresstest=";
+                        //char* createdataset_string = "createdataset="; Unused?
+                        //char* stresstest_string = "stresstest=";  Unused?
```

Comment:
How are those unused? They are used in line 945.

Reply:
It seems that our internal build instructions don't include the dbengine... I'll update the Travis pipeline again and fix this differently.

Reply:
I cannot reply underneath your other comment (github ui?)

> Which is not a unit test? In any case, -Wformat-signedness does not work with old compilers and can break with many distributions.

This is not changing how we package the software for distribution so it doesn't matter. This is how we check that incoming PRs compile on the known build environment inside the Travis pipeline.

Reply:
@knatsakis Please could you look at https://travis-ci.com/netdata/netdata/jobs/256057119 ?

Because we do not have libJudy installed the "standard build" on Travis is not our standard build. The installer disables the dbengine when Judy is not available, so we are not checking that our standard configuration builds.

Also it seems that CMocka is missing from that environment as well - so we could not enable the new unit tests.

How do we fix this?

Reply:
@amoss can you please create a new issue describing which dependencies you want me to add?

Reply:
I can, or if possible I could fix it here. Is it the install block in L19-L37 of the `.travis.yml` file? I can push a commit that adds the missing packages from https://docs.netdata.cloud/packaging/installer/ ?

I'm looking at the "This is how to do it by hand" section and the later " Netdata DB engine can be enabled when these are installed" sections.